src/lib/initOptions.ts

```diff
@@ -76,9 +76,9 @@ function initOptions(runOptions: RunOptions, { cli }: { cli?: boolean } = {}): O
     programError(options, chalk.red(`Cannot specify both --packageFile and --deep. --deep is an alias for --packageFile '${deepPatternPrefix}package.json'`))
   }
 
-  const target = options.newest ? 'newest'
+  const target: Target = options.newest ? 'newest'
     : options.greatest ? 'greatest'
-    : options.target || options.semverLevel || 'latest'
+    : options.target || (options as any).semverLevel || 'latest'
```

Comment:
I could type this as `{ semverLevel: string }` instead if preferred, but I figured since it's deprecated anyways it will probably go away soon.

Reply:
And actually if I type that properly then the check will fail because `string` is not assignable to type `Target | undefined`.  So leaving it as `any` to support the deprecated usage is easiest, but I can figure out another typing if needed.